src/http/raw/params.rs

```diff
@@ -146,14 +146,22 @@ enum IterInner<'a> {
     Array(std::slice::Iter<'a, serde_json::Value>),
 }
 
+#[derive(Debug)]
+pub enum Entry<'a> {
```

Comment:
Why can't this be in `common`?

Reply:
It is only used for the iterator out to distinguish between `No Params`, `Array` or `Object/Map` which is only is used by `raw::Params`. With that said I don't think we the iterator implementations anymore because it was only used for the debug implemenation but it might still be useful.

I think the intention was to have `common::Params` as serialize/deserialize and `raw::Params` as wrapper with some convince methods to simplify its use. I'm in favor of moving `raw::Params` to `common` but it couldn't come up with a good name for it, so left it to fix in another PR.